libminifi/test/unit/SocketTests.cpp

```diff
@@ -190,7 +190,7 @@ TEST_CASE("TestTLSContextCreation", "[TestSocket8]") {
     std::function<bool()> f_ex = createSocket;
     utils::Worker<bool> functor(f_ex, "id");
     std::future<bool> fut;
-    pool.execute(std::move(functor), fut);
+    pool.execute(std::move(functor), fut);  // NOLINT(bugprone-use-after-move)
```

Comment:
What kind of use-after-move is happening here? This looks perfectly valid to me, not reusing anything after move.

Reply:
I don't know either why it gives a warning, it seems okay to me too. I suspected that maybe the TEST_CASE macro does something I'm not aware of, but as it looked okay I only put the NOLINT comment there.

Reply:
If we get these false positives, I'd rather disable the check. This is clearly correct usage, and I don't want to have NOLINT everywhere.

Reply:
It would be better, but this false positive only occurs in test code so I would not disable it as it can catch some real problems in production code

Reply:
OK, fair point.